helm/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: coder
+  labels:
+    {{- include "coder.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.coder.service.type }}
+  ports:
+    - name: http
+      port: 80
+      targetPort: http
+      protocol: TCP
+    - name: https
+      port: 443
+      targetPort: https
```

Comment:
I don't think we change the deployment port we listen on, so we can't route to 443 here.

Reply:
we should probably fix that and provide a second listen address for https traffic. I'll open an issue

Reply:
https://github.com/coder/coder/issues/3123